src/client/reducers/chartTimePeriodIndex.js

```diff
@@ -2,16 +2,19 @@ import {
   SET_CHART_TO_DEFAULT_TIME_PERIOD,
   UPDATE_CHART_TIME_PERIOD_INDEX
 } from '../actions';
-import { INDEX_ONE_YEAR } from '../../constants/utilityConstants';
+import { CHART_META_DATA } from '../../constants/formatting';
+import { LABEL_ONE_YEAR } from '../../constants/userFacingStrings';
 
-const DEFAULT_TIME_PERIOD_INDEX = INDEX_ONE_YEAR;
+const defaultTimePeriodIndex = CHART_META_DATA.findIndex(element =>
```

Comment:
With this code, the app won't break if the index of the one year button ever changes.